apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -51,7 +53,7 @@ export default function RubricsContainer({unit, lesson}) {
           required={true}
           onChange={() => console.log('dropdown changed')}
         >
-          {tempLevelList.map(level => (
+          {levelsForDropDownList.map(level => (
             <option key={level} value={level}>
               {level}
```

Comment:
You may need to update these to be `level.id` for the value and `level.name` for the display